dev-util/codespell/codespell-2.2.1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} pypy3 )
```

Comment:
Please move to `PEP517` mode - [docs](https://projects.gentoo.org/python/guide/distutils.html#basic-use-pep-517-mode)